guides/common/modules/proc_converting-hosts-with-convert2rhel.adoc

```diff
@@ -0,0 +1,23 @@
+[id="converting-hosts-with-convert2rhel_{context}"]
+
+= Converting Hosts with Convert2RHEL
+
+Use the following procedure to perform the conversion.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts* *>* *All Hosts* and select the target hosts you want to convert.
+You can use the *Search* field to filter the host list.
+. From the *Select Action* list, select Schedule Remote Job.
+. On the *Job Invocation* page, define the main job settings.
+. In the *Job category* field, select Convert 2 RHEL.
+. Optional: In the *Job template* field, select Convert to RHEL.
+. Optional: In the *Bookmark* field, select the Bookmark.
```

Comment:
drop this step, it does not make sense in this case, people won't define a bookmark for hosts to be converted, they already grouped them in the host group in the procedure above

Reply:
The Bookmark step?

Reply:
yes, I'd drop the line I've attached the comment to

Reply:
Fixed.